doc/method/sources.md

```diff
@@ -1,77 +1,169 @@
+The following information might also go in the table: number of binomials,
+number of exclusive binomials (i.e. coming only from that source).
 
-*Index Fungorum*  
-We incorporated Index Fungorum to improve the coverage and classification of Fungi. We obtained database dumps of Index Fungorum around [date] (noting that the version of IF in in GBIF was [XXX]
-years older).  [reference]
+    irmng,1111550,313261
+    worms,258378,53233
+    silva,13953,44
+    gbif,1629523,486411
+    ncbi,360455,88358
+    if,237482,18291
+    addition,15,15
 
-We gave the Fungi branch of Index Fungorum higher priority than NCBI
-or GBIF.  We kept the non-fungal data at low priority
-(between NCBI and GBIF) in case it could provide any useful
-information.
+Maybe some measure of resolution, like maximum depth.
 
-*Hibbett 2007*
-The OpenTree Fungi curators provided am order-level
+end table]
+
+The choice and ranking of input taxonomies are driven by the three
+requirements listed in the introduction: 1) OTU coverage (mapping OTUs
+from phylogenies to taxa in the taxonomy); 2) a backbone hierarchy in the
+absence of phylogenies that cover a particular clade; and 3) taxonomic coverage
+of the tree of life beyond the OTUs present in the input trees. 
+
+As an open science project, Open Tree of Life only uses information
+sources that are not subject to terms of use (data use agreement).
+This policy decision imposes an additional requirement on the
+taxonomy, ruling out some sources that would otherwise be a good fit
+for the project.
+
+
+Details of sources: [in what order? the rank order is in the table,
+use rank order or something different?  historical? by number of
+taxa?]
+
+*NCBI taxonomy*  
+The first requirement of the taxonomy is to align OTUs across
+phylogenetic studies.  This need is largely met by using the NCBI
+taxonomy, since (1) most modern phylogenetic studies are molecular,
+(2) publishers require molecular sequences used in studies to be
+deposited in GenBank, and (3) every GenBank deposit is annotated with
+a taxon in the NCBI taxonomy.  NCBI taxonomy also tends to be more
+phylogenetically informed than other taxonomies (see Results, below),
+which makes it a good backbone for Open Tree's purposes. The NCBI
+taxonomy therefore forms the nucleus of OTT.
+
+However, as NCBI taxonomy only contains taxa that have sequence
+information, it only has 360455 records with standard binomial species
+names.  It therefore does not meet the taxonomic coverage requirement.
+
+The particular version of NCBI taxonomy used in OTT 2.10 was
+downloaded from NCBI on [date].  [maybe put dates in table??]
+
+
+*GBIF backbone taxonomy*  
+The GBIF backbone taxonomy provides taxonomic coverage - 1.6 million
+species (binomials).  The GBIF backbone draws from a number of sources
+(including IRMNG and Index Fungorum), has ongoing institutional
+support for maintenance and growth, and provides access without
```

Comment:
this reminds me I should list somewhere sources requested by curators but not yet added: paleobio DB, bird list, plant list, fishbase.  also must mention that we're prohibited from using IUCN, the only recent mammal list (maybe I already say that)